### PR TITLE
TGate -> Toffoli for signed to twos.

### PR DESCRIPTION
**Pull request by fdmalone**
Similar to #468 , use the more natural cost.

### CODE REVIEW

**Review by mpharrigan — DISMISSED** [review]
missing a key component of the change

**Comment by fdmalone** [comment]
Test?

File: qualtran/bloqs/arithmetic.py [hunk 1]
```diff
@@ -1142,4 +1142,4 @@ def signature(self) -> Signature:
     def build_call_graph(self, ssa: 'SympySymbolAllocator') -> Set['BloqCountT']:
         # Take the sign qubit as a control and cnot the remaining qubits, then
         # add it to the remaining n-1 bits.
-        return {(TGate(), 4 * (self.bitsize - 2))}
+        return {(TGate(), (self.bitsize - 2))}
```
Comment (tanujkhattar) [inline qualtran/bloqs/arithmetic.py:1145]: This was the suspense @fdmalone 
```suggestion
        return {(Toffoli(), (self.bitsize - 2))}
```

Comment (fdmalone) [reply qualtran/bloqs/arithmetic.py:1145]: phew, thanks! A test would have checked me :)

**Review by tanujkhattar — APPROVED** [review]

**Pull request merged**